oscal-data-repository-commons/src/main/java/com/easydynamics/oscal/data/marshalling/IterableAssemblyClassBinding.java

```diff
@@ -4,13 +4,29 @@
 import gov.nist.secauto.metaschema.binding.IBindingContext;
 import gov.nist.secauto.metaschema.binding.io.json.IJsonWritingContext;
 import gov.nist.secauto.metaschema.binding.model.DefaultAssemblyClassBinding;
+import gov.nist.secauto.oscal.lib.model.ImplementedRequirement;
 import java.io.IOException;
+import java.util.Map;
+import javax.xml.namespace.QName;
+import org.apache.commons.collections4.map.HashedMap;
 
 /**
  * Extends DefaultAssemblyClassBinding to add writing of root items as an array.
  */
 public class IterableAssemblyClassBinding extends DefaultAssemblyClassBinding {
 
+  /**
+   * Map of objects that are not declared as root in liboscal-java,
+   * but we want to treat them as such.
+   */
+  private static final Map<Class<?>, QName> secondaryRootObjects;
+
+  static {
+    secondaryRootObjects = new HashedMap<>();
+    secondaryRootObjects.put(ImplementedRequirement.class,
+        new QName("implemented-requirement"));
+  }
+
```

Comment:
Because we only ever use the interface here, we never write to it after building, and we're Java 11+, we could potentially just use [`Map.ofEntries`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html#ofEntries(java.util.Map.Entry...)). But if there's a specific reason to use `HashedMap`, this may not work for us.

I am also pretty biased against initializers.

```suggestion
  private static final Map<Class<?>, QName> secondaryRootObjects = Map.ofEntries(
    Map.entry(ImplementedRequirement.class, new QName("implemented-requirement"),
  );
```

Reply:
Addressed.